elastic/kibana
# Remove `alerting_framework_heath` from the alerting framework health API response

Resolves https://github.com/elastic/kibana/issues/135035.

In this PR, I'm removing the deprecated attribute `alerting_framework_heath` from the alerting framework health API.

## To verify

1. Start Kibana.
2. Call `/api/alerting/_health`.
3. Notice `alerting_framework_heath` is no longer part of the HTTP API response.
